app/scripts/views/mixins/notifications-mixin.js

```diff
@@ -0,0 +1,117 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * A view mixin to interface with the notifications channel
+ *
+ * Any registered handlers are automatically unregistered
+ * when the view is destroyed.
+ */
+
+define([
+], function () {
+  'use strict';
+
+  var NotificationsChannelMixin = {
+    notificationsInitialize: function (options) {
```

Comment:
I'm not a massive fan of the name `notificationsInitialize`, or of `notificationsOn`, `notificationsOff`, `notificationsOffAll` and `notificationsClear`. They read like written by Yoda, they were.